pytest.ini

```diff
@@ -1,7 +1,7 @@
 [pytest]
 python_files=test_*.py
 python_functions=nothing matches this pattern
-addopts = --durations=30 --tb=native
+addopts = --durations=30 --tb=native -n 1 --dist=loadfile
```

Comment:
That's an interesting point.  The `setup.py` requires `xdist`, so I expect it to be there.  Nick also said that it is a requirement of the test env, so I would be okay expecting it.

Because of how PyTest intercepts logging, it's pretty hard to test safely without:
 
1. Using `pytest-xdist` over `pytest-parallel` (I just go lucky there).
2. And forcing `--dist=loadfile`, which has to be done at the `pytest.ini` level, not test-by-test.

I suppose I felt I need the new `reactors.py` test to prove the new system is working. If we don't feel comfortable with this change right here... I guess my suggestion would be to delete that new `reactors.py` test. 

Which path do you like better: (a) keep this line, or (b) remove the one test that demands it?

Reply:
Oh, sorry i didnt know that we had `xdist` in the `dev` extra. Disregard!